src/main/java/pl/edu/pk/cosmo/cookbookbackend/controller/request/SaveAccountRequest.java

```diff
@@ -4,11 +4,13 @@
 
```

Comment:
Mam wrażenie że usunąłeś wszystkie obiekty Example z przykładowego brancha. Mógłbyś je wrzucić z powrotem?

Reply:
Chodzi o default? Nie wprowadzałem tam zmian, a jak właśnie sprawdziłem to chyba są